What went wrong

root@iZm5ehpdvi5okfb628x3icZ:FullNode/java-tron-Odyssey-v3.6.6# ./gradlew build

FAILURE: Build failed with an exception.

* Where:
Build file '/usr/local/wallet/trx/FullNode/java-tron-Odyssey-v3.6.6/build.gradle' line: 336

* What went wrong:
Could not determine the dependencies of task ':buildDBConvertJar'.
> Could not resolve all files for configuration ':compile'.
   > Could not resolve io.grpc:grpc-core:[1.14.0].
     Required by:
         project : > io.grpc:grpc-netty:1.14.0
      > Could not resolve io.grpc:grpc-core:[1.14.0].
         > Failed to list versions for io.grpc:grpc-core.
            > Unable to load Maven meta-data from http://repo.spring.io/plugins-release/io/grpc/grpc-core/maven-metadata.xml.
               > Could not HEAD 'http://repo.spring.io/plugins-release/io/grpc/grpc-core/maven-metadata.xml'. Received status code 403 from server: Forbidden
   > Could not resolve io.netty:netty-codec-http2:[4.1.27.Final].
     Required by:
         project : > io.grpc:grpc-netty:1.14.0
      > Could not resolve io.netty:netty-codec-http2:[4.1.27.Final].
         > Failed to list versions for io.netty:netty-codec-http2.
            > Unable to load Maven meta-data from http://repo.spring.io/plugins-release/io/netty/netty-codec-http2/maven-metadata.xml.
               > Could not HEAD 'http://repo.spring.io/plugins-release/io/netty/netty-codec-http2/maven-metadata.xml'. Received status code 403 from server: Forbidden

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output.

* Get more help at https://help.gradle.org

BUILD FAILED in 5s

--- a/build.gradle
+++ b/build.gradle
@@ -1,6 +1,6 @@
 allprojects {
     version = "1.0.0"
-    apply plugin: "java"    
+    apply plugin: "java"
 }
 
 subprojects {
@@ -29,7 +29,7 @@ subprojects {
     repositories {
         mavenLocal()
         mavenCentral()
-        maven { url 'http://repo.spring.io/plugins-release' }
+        maven { url 'https://repo.spring.io/plugins-release' }
         maven { url 'https://jitpack.io' }
     }
 
@@ -68,4 +68,4 @@ task copyToParent(type: Copy) {
     }
 }
 
-build.finalizedBy(copyToParent) 
+build.finalizedBy(copyToParent)

